src/components/handbook-navigation/intra-page.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Link } from "gatsby";
+import styled from "styled-components";
+// import { content } from "./content";
+
+const JoinCommunityWrapper = styled.div`
+  .intra-page {
+    position: sticky;
+    top: 10rem;
+    right: 0rem;
+    margin-right: 1rem;
+    padding-bottom: 5rem;
+    align-items: left;
+    justify-content: space-around;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+
+    .active {
+      font-weight: 5000;
+      color: ${(props) => props.theme.secondaryColor};
+    }
+
+    ul {
+      list-style: none;
+      top: 3rem;
+    }
+    li {
+      padding-bottom: 0.5rem;
+      padding-top: 0.5rem;
+    }
+    @media only screen and (max-width: 900px) {
+      width: 0;
+      opacity: 0;
+      height: 0;
+      transition: none;
+      visibility: hidden;
+    }
+  }
+
+  .learn {
+    width: 10rem;
+  }
+`;
+
+function IntraPage({ contents }) {
+  return (
+    <JoinCommunityWrapper>
+      <div className="intra-page">
+        <ul>
+          {contents.map((x) => (
+            <li key={x.id}>
+              <Link to={x.link} key={x.id} activeClassName="active">
```

Comment:
Change the above line to :
```
<Link to={`${window.location.pathname}${x.link}`} key={x.id} activeClassName="active">
```

This will fix the issue that @warunicorn19 mentioned.

Reply:
@toth2000 That works thnx

Reply:
@toth2000 Hey error is showing up in the build, that window is not defined , can you suggest a solution.